### ISSUE TITLE
CMS: Input fields with operator dropdowns do not have associated labels

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12554]
## Description
**Still an issue in D10**

When adding filters to views, there is an option to add a fieldset filter combo of an operator dropdown plus an input field. When this is added, this input field does not have a label element associated with it:
![image](https://user-images.githubusercontent.com/106678594/217922840-91466984-137c-4faf-807a-13004fd5afb9.png)
(Screenshot is from the Documentation list: Structure Menu > [Content Model Documents](https://staging.cms.va.gov/admin/structure/cm_document))

### Additional Info
There is likely an existing issue on drupal.org issue but it may take some digging to find it. Here is the appropriate issue queue
https://www.drupal.org/project/issues/drupal?component=views.module

From Swirt: A little quick fishing expedition turned up nothing... but it could take a little more finesse on the search https://www.drupal.org/project/issues/drupal?text=filter+fieldset&component=views.module

From Laura: I also looked and was able to find 2 somewhat related issue but I do not believe these are  exactly what I'm referring to:
- https://www.drupal.org/project/drupal/issues/2096013
- https://www.drupal.org/project/drupal/issues/3120627

## Acceptance Criteria
- [ ] Update fields to be able to add labels
- [ ] Accessibility lead review

### Team
Please check the team(s) that will do this work.

- [x] `CMS Team`
- [ ] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`

## Previous Team Points
8

**Comment by ndouglas** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Accessibility-Issues-62b4be010177900017e82254/issues/department-of-veterans-affairs/va.gov-cms/12554?planning-poker) with Zenhub @edmund-dunn @tonytaylor

**Comment by srancour** [comment]
Still an issue and should be fixed.
